var/spack/repos/builtin/packages/py-openmc/package.py

```diff
@@ -31,7 +33,7 @@ class PyOpenmc(PythonPackage):
     variant("mpi", default=False, description="Enable MPI support")
```

Comment:
I would remove this variant, it doesn't do anything in the package. If users want MPI support, they can do `spack install py-openmc ^openmc+mpi ^hdf5+mpi`.

Reply:
One question regarding this -- how would I handle the dependency to `py-mpi4py`? Right now, when the `+mpi` variant is added, it picks up the `py-mpi4py` dependency. I'm not sure how to do that in the absence of the `mpi` variant within `py-openmc`.

Reply:
Oh you're right, in that case I would keep the variant.